ExploreASL_Initialize.m

```diff
@@ -600,6 +600,33 @@ function xASL_init_AddDirsOfxASL(MyPath)
                 end
             end
         
+			% Check and remove empty spaces at the start and end of the DatasetRoot
```

Comment:
We have functions for this already, e.g., `xASL_adm_ReplaceSymbols` or `xASL_adm_UnixPath`, but they don't do exactly what you want here I believe.

I would at least make this a separate section in the code, or even a subfunction, to keep it all readable.

This could be as easy as replacing % with %%

Reply:
ReplaceSymbols and UnixPath indeed does a different thing.
I'm adding %%. Fine?